public/app/features/dashboard-scene/settings/variables/components/QueryVariableForm.tsx

```diff
@@ -40,12 +41,11 @@ interface QueryVariableEditorFormProps {
 }
 
 export function QueryVariableEditorForm({
-  datasource,
+  datasource: datasourceRef,
```

Comment:
I would rename the prop to dsRef instead to make it more clear (and not have to rename prop locally here)

Reply:
We're using `datasource` for other different components(`QueryEditorRows`, `AdHocFilterValue`,`AdHocFilterKey`,`AdHocFilterEditor`), including the `QueryVariableModel`, this is why I decided to use `datasource`. We're using [the naming/renaming](https://github.com/search?q=repo%3Agrafana%2Fgrafana%20datasource%3A%20datasourceRef%2C&type=code) in different places as well.

This is why I preferred `datasource`.